README_new.md

```diff
@@ -0,0 +1,89 @@
+# pyprobml
+
+<img src="https://img.shields.io/github/stars/probml/pyprobml?style=social">
+
+
+Python 3 code to reproduce the figures in the book series [Probabilistic Machine Learning](https://probml.github.io/pml-book/) by Kevin Patrick Murphy.
+This is work in progress, so expect rough edges!
+(Some demos use code from our companion [JAX State Space Library](https://github.com/probml/JSL).)
+ 
+
+## Running the notebooks
+
+The notebooks needed to make all the figures are available at the following locations.
+
+* [Volume 1 figure notebooks](https://github.com/probml/pyprobml/tree/master/notebooks/book1)
+* [Volume 2 figure notebooks](https://github.com/probml/pyprobml/tree/master/notebooks/book1). (Note: volume 2 is not finished yet.)
+
+Notebooks are saved in chapter-wise folders. For example, a notebook for figure 2.3 is saved in the folder `notebooks/book1/02/`.
+
+In addition to the figure notebooks, there are a series of manually created notebooks, which create additional figures, and provide supplementary material for the book. These are stored in the [`misc` folder](https://github.com/probml/pyprobml/tree/master/notebooks/misc).
+
+### Running notebooks in colab
+
+[Colab](https://colab.research.google.com/notebooks/intro.ipynb) has most of the libraries you will need (e.g., scikit-learn,  JAX) pre-installed, and gives you access to a free GPU and TPU. We have a created a [intro to colab](https://colab.research.google.com/github/probml/probml-notebooks/blob/main/notebooks/colab_intro.ipynb) notebook with more details. To run the notebooks on colab in any browser, you can go to a particular notebook on GitHub and change the domain from `github.com` to `githubtocolab.com` as suggeted [here](https://stackoverflow.com/a/67344477/13330701). If you are using Google Chrome browser, you can use ["Open in Colab" Chrome extension](https://chrome.google.com/webstore/detail/open-in-colab/iogfkhleblhcpcekbiedikdehleodpjo) to do the same with a single click.
+
+## Running the noteboks locally 
+
+We assume you have already installed [JAX](https://github.com/google/jax#installation) and
+[Tensorflow](https://www.tensorflow.org/install) and [Torch](https://pytorch.org/),
+since the details on how to do this depend on whether you have a CPU, GPU, etc.
+
+You can use any of the following options to install the other requirements.
+
+* Option 1
+
+```bash
+pip install -r https://raw.githubusercontent.com/probml/pyprobml/master/requirements.txt
+```
+
+* Option 2
+
+Download [requirements.txt](https://github.com/probml/pyprobml/blob/master/requirements.txt) locally to your path and run
+
+```bash
+pip install -r requirements.txt
+```
+
+## Running scripts for vol 2
```

Comment:
Remove this section on 'scripts for vol 2'. (If there are any notebooks that need JSL, they can install it locally for that colab.)